server/games/game.py

```diff
@@ -795,6 +799,26 @@ def report_army_stats(self, stats_json):
         self._army_stats_list = json.loads(stats_json)["stats"]
         self._process_pending_army_stats()
 
+    def is_visible_to_player(self, player: Player) -> bool:
+        if player == self.host:
```

Comment:
so u took care of this I see

Reply:
To avoid funny glitches probably should also allow everybody that is in the game

Reply:
Yea that’s a good point, the same thing could happen if the host unfriends you while you’re in lobby.

Reply:
Does that mean I can currently glitch people if I host a game and then add the people who joined as foes while they are in the game lobby? Of course I don't know what "glitch" means in that context - would the game start?
Either way, axel's suggestion below would resolve that, too.

Reply:
Hilariously enough, actually no because when you add/remove a friend or foe, the database is updated, but the inmemory data in `player.foes` and `player.friends` is not updated, so the server will still consider you to be friends/foes with that person until you log out and back in :P. That's another thing I'm fixing in this PR.

Reply:
When I said "glitch" then I was not sure what the effects are. I also can not tell but I am 100% sure nobody expected such a thing when writing the client code.